api/go.mod

```diff
@@ -3,9 +3,9 @@
 
 module github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/api
 
-go 1.22.0
+go 1.24
```

Comment:
on CAPX api module we tend to stick to one version before the latest (oldest supported Go version) just in case people are importing structs as a package.

Reply:
Good call out: reverted language compatibility to 1.23 while still using 1.24.1 for building.